magma/tests/lib.rs

```diff
@@ -1,5 +1,5 @@
 #![no_std]
-#![cfg_attr(rustfmt, rustfmt_skip)]
```

Comment:
This appears to have no effect.

See also: https://github.com/rust-lang/rustfmt/issues/4028

Reply:
It does disable rustfmt in the test function. I would prefer to keep data there 16 bytes per line.

Reply:
Ok. I'd suggest using the new nested attribute syntax:

```rust
#![rustfmt::skip]
```

Reply:
Oh whoops, that isn't stable for inner attributes yet :cry:

Reply:
Looks like this feature is still unstable.